include/crpropa/module/AdiabaticCooling.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
```

Comment:
USe include guards instead of pragma once for consistency in CRPropa and to avoid potential compiler issues